### PR TITLE
Touch up in introduction section

### PR DESCRIPTION
**Pull request by squell**
@hugovandepol and I had a go at the intro section, to link to the hiring page, add licensing info, and other changes.

### CODE REVIEW

File: slides/1_1-introduction.md [hunk 1]
```diff
@@ -107,10 +105,11 @@ layout: with-footer
...
 # A quick note about Tweede golf
 
-- **Leading Rust agency in NL** Running Rust in production since 2017
-- **Working on Rust adoption** Through both teach-rs and in-company trainings
-- **Working on Rust projects** Like Rust implementations of PTP, NTP, sudo and su
-- **Active community member** Organizing RustNL 2023 and active in various Rust meetups
+- **Leading Rust experts** Running Rust in production since 2017
+- **Engineers** Web, systems, embedded software development
+- **Open source** Impactful implementations of Network Time Protocol, sudo
+- **Adoption** Through both teach-rs and in-company trainings
```
Comment (LisetRoos) [inline slides/1_1-introduction.md:111]: 111: 'training' (singular) instead of 'trainings'

Comment (LisetRoos) [reply slides/1_1-introduction.md:111]:
```suggestion
- **Adoption** Through both teach-rs and in-company training
```

File: book/src/tweedegolf-logo.png
Comment (tdittr) [inline book/src/tweedegolf-logo.png:1]: Can we just link this? Adding it to the repository makes it bigger and slower to clone.

Comment (squell) [reply book/src/tweedegolf-logo.png:1]: It's 19 kilobytes, but granted that's 25% of the mdbook. Let's see if I can find a proper link ;-)

Comment (squell) [reply book/src/tweedegolf-logo.png:1]: Haven't found a link on our website yet. But I can replace it with the SVG

Comment (squell) [reply book/src/tweedegolf-logo.png:1]: Using SVGO, I've also reduced that SVG further in size down to 2kb where it is no burden on the repository.  I can also reduce it down to 600 bytes but then it'll start looking like it's a still from Day of the Tentacle.

Comment (tdittr) [reply book/src/tweedegolf-logo.png:1]: Nice! we just need to squash the changes now so it is also no longer in the history

**Review by tdittr — APPROVED** [review]
Looks good to me now! Feel free to "Squash and merge"

**Pull request merged**